src/Chapter04/Listing04.22.NestedIf.cs

```diff
@@ -0,0 +1,48 @@
+// Justification: Checkihng for null isn't discussed yet.
+#pragma warning disable CS8604 // Possible null reference argument
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter04.Listing04_22
+{
+    public class TicTacToeTrivia
+    {
+        public static void Main()
+        {
+            #region INCLUDE
+            int input;    // Declare a variable to store the input
+
+            Console.Write(
+                "What is the maximum number " +
+                "of turns in tic-tac-toe?" +
+                " (Enter 0 to exit.): ");
+
+            // int.Parse() converts the ReadLine()
+            // return to an int data type
+            input = int.Parse(Console.ReadLine());
+
+            // Check 1.
```

Comment:
```suggestion
            // Condition 1.
```

nit: keep verbiage the same throughout listing